boards/cc3200-launchxl/dist/flasher.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+TOOL_DIR=$1
+PORT=$2
+ELFILE=$3
+
+# change folder since cc3200prog expects dlls in its pwd
```

Comment:
to me, `pwd` is the name of a program in coreutils.
i assume you mean working directory.